.github/workflows/pragmaonce.yml

```diff
@@ -0,0 +1,30 @@
+name: pragma-once checker
+
+on: [pull_request_target]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+        persist-credentials: false
+        # We need the history of the dev branch all the way back to where the PR
+        # diverged. We're fetching everything here, as we don't know how many
+        # commits back that point is.
+        fetch-depth: 0
+
+    - name: Install prerequisites
+      env:
+        DEBIAN_FRONTEND: noninteractive
+      run: |
+        sudo apt update -y
+        sudo apt install -y git
```

Comment:
According to [the docs](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#tools), git is already installed in the container, so we don't need this step at all -- removing it should speed up the check.